### PR TITLE
JBEAP-8362 - TransformerFactory doesn't propogate SECURE_PROCESSING feature to DocumentBuilderFactory/SAXParser it creates

### PR DESCRIPTION
**Pull request by rpelisse**
Issue: https://issues.jboss.org/browse/JBEAP-8362
Upstream Issue (sort of): https://issues.jboss.org/browse/WFCORE-4023

### CODE REVIEW

File: src/org/apache/xerces/jaxp/DocumentBuilderFactoryImpl.java [hunk 1]
```diff
@@ -76,7 +76,8 @@
     /**
      * State of the secure processing feature, initially <code>false</code>
```
Comment (kabir) [inline src/org/apache/xerces/jaxp/DocumentBuilderFactoryImpl.java]: The javadoc should probably say false now?

File: src/org/apache/xerces/jaxp/SAXParserFactoryImpl.java [hunk 2]
```diff
@@ -61,7 +61,7 @@
     /**
      * State of the secure processing feature, initially <code>false</code>
```
Comment (kabir) [inline src/org/apache/xerces/jaxp/SAXParserFactoryImpl.java]: true?

**Comment by rpelisse** [comment]
duh! Yes, of course it should (gosh this is exactly why I'm against Javadocs comment - the guy from Clean Code was right!). Let me fix that right away!

**Comment by rpelisse** [comment]
@kabir fixed! Let me know if you see anything else that may cause trouble.

**Pull request merged**